packages/tables/resources/views/index.blade.php

```diff
@@ -24,7 +24,7 @@
     $filterIndicators = $getFilterIndicators();
     $hasColumnGroups = $hasColumnGroups();
     $hasColumnsLayout = $hasColumnsLayout();
-    $hasSummary = $hasSummary();
+    $hasSummary = $hasSummary($this->getAllTableSummaryQuery());
```

Comment:
@danharrin Same for this line, the `getAllTableSummaryQuery ` method is trying to return null when using custom data.